### DISCUSSION TITLE
v51.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#4803] [Release Discussions]
<img  width="260" align="left" src="https://user-images.githubusercontent.com/4661658/215844472-67489786-bb31-40d5-937e-bd497a3872d1.jpg"><img width="260"  align="left" src="https://user-images.githubusercontent.com/4661658/215844510-eae189f9-cdbe-4936-ad08-96e33f22a11a.jpg">

  <h3>Standalone measuring app</h3>
<a href="https://play.google.com/store/apps/details?id=de.westnordost.streetmeasure"><img width="128" src="https://raw.githubusercontent.com/streetcomplete/StreetMeasure/master/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png"></a>
  <p>Measuring distances with the camera has been outsourced into a standalone app, so you can also measure things outside of the context of solving quests: <a href="https://play.google.com/store/apps/details?id=de.westnordost.streetmeasure">StreetMeasure</a></p>
  <p>This has been done for licensing reasons (see #4289). In a nutshell, ARCore (= <a href="https://play.google.com/store/apps/details?id=com.google.ar.core">Google Play Services for AR</a>), on which the measuring is dependent on, is not open source.<br>
     This is why it is also not available on the main repo in F-Droid. If you do not have Google Play, you can <a href="https://github.com/streetcomplete/StreetMeasure/releases">download the APK from GitHub</a>, but then you also need to install ARCore.</p>
  <h3>New Quests</h3>
  <ul>
  <li><i>"Is this also here outside of winter?"</i>, asked for grit bins (#4749, #4754), by @FloEdelmann</li>
  </ul>
  <h3>Enhancements</h3>
  <ul>
  <li>The standalone measure app now explains how to achieve optimal precision and warns if the potential for a measuring error becomes too big (#4655)</li>
  <li>Parking overlay: Support new street parking schema (#4664, #4768)</li>
  <li>Parking overlay: Add option to specify that cars park on alternating sides (i.e. not enough space to park on both sides at the same time)</li>
  <li>Address overlay: Make entering addresses in places with no street names more efficient (#4718, #4751), by @Helium314</li>
  <li>Ask for kerbs also on traffic islands (#3786), by @arrival-spring</li>
  <li>Do not ask various quests for ways that only exist to connect an e.g. footway to the street-way (#4787)</li>
  <li>Notes: You can now preview the to be attached photos by tapping on them (#2497, #4797), by @Helium314</li>
  <li>The name and location of a place is now also shown in the undo dialog (#4767, #4772), by @Helium314</li>
  <li>Other small enhancements, improved wordings, ... (#4762, #4750, #2880, #4765, #4710, #4781, #4745, #4800...), thanks @matkoniecz, @FloEdelmann, @Helium314, @dbdean</li>
  </ul>
  <h3>Fixes</h3>
  <ul>
  <li>Don't ask whether vacant shop is still the same shop (#4752, #4756), by @Helium314</li>
  <li>(Street) names should not automatically be capitalized in Georgian (#4784, #4785), by @balsoft</li>
  <li>Other fixes (#4774, #4763, #4790...)</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v51.0'>v51.0</a>.</em>

**Comment by MMK21Hub** [comment]
Is this release not available on F-Droid yet? I suppose there must be a lag time because the app has to be compiled separately for F-Droid, but I've never noticed a delay before.

**Reply by Helium314** [reply]
This delay has always existed. Usually it's a few days, sometimes more than a week.

**Reply by matkoniecz** [reply]
At one point it was big enough to be measured in months.

**Reply by MMK21Hub** [reply]
Okay, that's good to know. Thanks for the info!
